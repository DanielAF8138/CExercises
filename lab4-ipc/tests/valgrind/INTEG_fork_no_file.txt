==3308131== Memcheck, a memory error detector
==3308131== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==3308131== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==3308131== Command: ../ipc cksum ../data/f4.txt
==3308131== 
cksum: ../data/f4.txt: No such file or directory
cksum(../data/f4.txt) = ''
==3308131== 
==3308131== HEAP SUMMARY:
==3308131==     in use at exit: 0 bytes in 0 blocks
==3308131==   total heap usage: 3 allocs, 3 frees, 8,293 bytes allocated
==3308131== 
==3308131== All heap blocks were freed -- no leaks are possible
==3308131== 
==3308131== For lists of detected and suppressed errors, rerun with: -s
==3308131== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
