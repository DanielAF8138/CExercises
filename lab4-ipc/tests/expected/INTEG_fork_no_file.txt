cksum(../data/f4.txt) = ''
